debug.xml

```diff
@@ -488,4 +488,8 @@
     <entry key='smokey.port'>5125</entry>
     <entry key='extremtrac.port'>5126</entry>
 
+    <entry key='arnavibin.port'>5127</entry>
+    <entry key='arnavibin.resetDelay'>60</entry>
```

Comment:
Please don't include `resetDelay`. Also, `resetDelay` is deprecated name, you should use `timeout`.